test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs

```diff
@@ -642,6 +642,20 @@ public void ItMigratesAndBuildsLibrary(string projectName)
         }
 
         [Fact]
+        public void ItMigratesAndBuildsAppWithExplicitInclude()
```

Comment:
I agree with unit test but I also think that we should have E2E (especially it is not too slow). If you strongly feel we should remove it I will remove it